### PR TITLE
Fix object literals

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by sjrd** [comment]
Since this is an easy one, and @nicolasstucki has officially started working on the project:
Review by @nicolasstucki 

Hint: there *is* something to blame me for (caught by my own review, but I left it there intentionally for the "exercise" ^^)

File: test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/DynamicTest.scala [hunk 1]
```diff
@@ -101,6 +101,8 @@ object DynamicTest extends JasmineTest {
       expect(x.bar).toEqual("foobar")
       expect(x.unknown).toBeUndefined()
 
+      obj(plop = 3)
```
Comment (nicolasstucki) [inline test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/DynamicTest.scala]: This should not be there.

File: test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/DynamicTest.scala [hunk 2]
```diff
@@ -111,6 +113,11 @@ object DynamicTest extends JasmineTest {
       expect(obj().anything).toBeUndefined()
     }
 
+    it("object literal in statement position - #1627") {
+      // Just make sure it does not cause a SyntaxError
+      js.Dynamic.literal(foo = "bar")
```
Comment (nicolasstucki) [inline test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/DynamicTest.scala]: Should also be tested without parameters.

**Comment by nicolasstucki** [comment]
Thats all.

File: test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/DynamicTest.scala (hunk 1 above)
Comment (sjrd) [inline test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/DynamicTest.scala]: Indeed ^^

File: test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/DynamicTest.scala (hunk 2 above)
Comment (sjrd) [inline test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/DynamicTest.scala]: Done on the next line.

**Comment by sjrd** [comment]
Updated.

**Comment by nicolasstucki** [comment]
LGTM

**Pull request merged**